sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -272,6 +272,31 @@ const BuildingDetails = ({
           />
         </GridCell>
       </GridSection>
+      <GridSection columns={3}>
+        <GridCell>
+          <Field
+            label={t("application.contact.county")}
+            name={"buildingAddress.county"}
+            id={"buildingAddress.county"}
+            placeholder={t("application.contact.county")}
+            error={
+              !!getAddressErrorMessage(
+                "buildingAddress.county",
+                fieldMessage(errors?.buildingAddress?.county)
+              )
+            }
+            errorMessage={getAddressErrorMessage(
+              "buildingAddress.county",
+              fieldMessage(errors?.buildingAddress?.county)
+            )}
+            inputProps={{
+              onChange: () => clearErrors("buildingAddress"),
+            }}
+            register={register}
+          />
+          <p className="field-sub-note">{t("listings.requiredToPublish")}</p>
```

Comment:
I don't think we actually validate that this is required anywhere -- i.e. you can save without the county field

Reply:
Agreed. I wasn't sure how/where to do that. It didn't seem like it was already happening and I hadn't yet figured out how to do that. 

Btw...have you tried publishing a listing yet? From my experience it doesn't point you to which fields aren't filled in, but are required ... and even when I ensured I filled in ones that had the label "required", it wasn't evident. Not a great user experience, but not sure if I'm missing something altogether.